src/cbc_sdk/platform/policies.py

```diff
@@ -0,0 +1,1222 @@
+#!/usr/bin/env python3
```

Comment:
The Policy class doesn't appear in read the docs.
I've configured rtd to build of this branch, so changes can be checked here:
https://carbon-black-cloud-python-sdk.readthedocs.io/en/cbapi-3641_policy_simplification

the file docs/cbc_sdk.platform.rst needs this section added (alphabetically):

cbc\_sdk.platform.policies module
----------------------------------

.. automodule:: cbc_sdk.platform.policies
   :members:
   :undoc-members:
   :show-inheritance:

Reply:
Got that, thanks.  I should probably also check for anything that needs removing from `cbc_dsk.endpoint_standard.rst`.